src/pages/faq.tsx

```diff
@@ -0,0 +1,65 @@
+import { Collapse, Text } from '@geist-ui/core';
+import { useRouter } from 'next/router';
+import { ReactMarkdown } from 'react-markdown/lib/react-markdown';
+
+const FAQ = () => {
+
+  const FAQquestions = {
```

Comment:
Why nest FAQs in FAQquestions when FAQs can be the top nest level? That's an extra level of unused nesting. Maybe this will have some usage in the future?

Reply:
FAQ = Frequently Asked Questions
FAQquestions = Frequently Asked Questions question

pure logic

Reply:
Yeah, because it can store more than 1 faq, so i think how about an array with name of faq ? And then questions in it ?

Reply:
Sorry, i didn't noticed it while converting from yaml